dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java

```diff
@@ -192,18 +193,19 @@ public ResponseEntity<?> scale(
     };
   }
 
-  private ResponseEntity<?> scaleBrokers(final List<Integer> ids, final boolean dryRun) {
+  private ResponseEntity<?> scaleBrokers(
+      final List<Integer> ids, final boolean dryRun, final boolean force) {
     try {
+      final ScaleRequest scaleRequest =
+          new ScaleRequest(
+              ids.stream().map(String::valueOf).map(MemberId::from).collect(Collectors.toSet()),
+              dryRun);
+      // here we assume, if it force request it is always force scale down. The coordinator will
+      // reject the request if that is not the case.
       final var response =
-          requestSender
-              .scaleMembers(
-                  new ScaleRequest(
-                      ids.stream()
-                          .map(String::valueOf)
-                          .map(MemberId::from)
-                          .collect(Collectors.toSet()),
-                      dryRun))
-              .join();
+          force
+              ? requestSender.forceScaleDown(scaleRequest).join()
```

Comment:
:question: Just do double check: `forceScaleDown` properly supports `dryRun`?

Reply:
Yes. It follows the same approach as other operations.